Animations/Animations.js

```diff
@@ -0,0 +1,194 @@
+// 'use strict';
+var React = require("react/addons");
+var Arrival = require('Arrival');
+var ReactTransitionGroup = React.addons.TransitionGroup;
+
+//makeTransition FUNCTION: creates animation for wrapped component based on props of wrapper component
+  //@param displayName STRING: intended name of the wrapper animation component
+  //@param transitions OBJECT: properties that define the default animation properties for the animation component wrapper
+  //Required keys:
```

Comment:
The keys listed in Required keys and Additional keys are keys on the React.createClass input object. Recommend moving these comments into that object and matching it with the appropriate keys.